graylog2-web-interface/src/views/components/datatable/DataTable.tsx

```diff
@@ -128,6 +163,32 @@ const DataTable = ({
     return Promise.reject();
   }, [config, widget, editing, formContext]);
 
+  const togglePin = useCallback((field: string) => {
+    const dirty = formContext?.dirty;
+
+    const updateWidget = () => {
+      const curVisualizationConfig = widget.config.visualizationConfig || DataTableVisualizationConfig.create([]).toBuilder().build();
```

Comment:
We can use nullish coalescing to make this a bit more idiomatic:
```suggestion
      const curVisualizationConfig = widget.config.visualizationConfig ?? DataTableVisualizationConfig.create([]).toBuilder().build();
```